controllers/ControllerAuth.php

```diff
@@ -3,123 +3,120 @@
+		(new AuthManager())->registerUser($params);
+		echo $this->getTwig()->render('login.twig', ['valid' => TRUE,
+		                                         'server' => $this->getServer()]);
 
 	}
 
-	public function checkAuth(){
-		$params = $this->_params;
-		$user = $this->checkLogin($params);
-		$key = 'user';
+	/**
+	 * Permet de vérifier les identifiants envoyé par l'utilisateur.
+	 */
+	public function checkAuth(): void {
+		$params = $this->getPost();
+		$user = (new AuthManager())->checkLogin($params);
 
 		if ($user === false){
-			echo $this->_twig->render('login.twig', ['notValid' => TRUE,
-			                                         'server' => $this->_server]);
+			echo $this->getTwig()->render('login.twig', ['notValid' => TRUE,
+			                                         'server' => $this->getServer()]);
 			exit();
 		}else{
-			( new Session )->setKey($key, $user);
-			header("Location: http://".$this->_server."/Home");
+			( new Session )->setKey('user', $user);
+			header("Location: http://".$this->getServer());
 		}
-
-
 	}
 
-	public function check(){
-		$params = $this->_params;
+	/**
+	 * Permet de vérifier les information envoyé par $_POST.
+	 */
+	public function check(): void {
+		$params = $this->getPost();
+		$userManager = new AuthManager;
 
 		foreach ($params as $key => $value){
 
 			if ($value != preg_replace('/\s+/', '', $value)){
-				echo $this->_twig->render('register.twig', ['space' => TRUE,
-				                                            'server' => $this->_server,
+				echo $this->getTwig()->render('register.twig', ['space' => TRUE,
+				                                            'server' => $this->getServer(),
 				                                            'key' => $params]);
 				exit();
 			}
 
 			if (strlen($value) < 3){
-				echo $this->_twig->render('register.twig', ['noValid' => TRUE,
-				                                            'server' => $this->_server,
+				echo $this->getTwig()->render('register.twig', ['noValid' => TRUE,
+				                                            'server' => $this->getServer(),
 				                                            'key' => $params]);
 				exit();
 			}
 		}
 
 		if($params['password'] != $params['confirme']){
-			echo $this->_twig->render('register.twig', ['valid' => TRUE,
-			                                            'server' => $this->_server,
+			echo $this->getTwig()->render('register.twig', ['valid' => TRUE,
+			                                            'server' => $this->getServer(),
 			                                            'key' => $params]);
 			exit();
 		}
 
-		if ($this->checkUser($params['pseudo'])){
-			echo $this->_twig->render('register.twig', ['noPseudo' => TRUE,
-			                                            'server' => $this->_server,
+		if ($userManager->checkUser($params['pseudo'])){
+			echo $this->getTwig()->render('register.twig', ['noPseudo' => TRUE,
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)